web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx

```diff
@@ -153,6 +171,23 @@ export function SendForm(props: SendFormProps) {
     <Spinner size={12} customClass="inline-block" />
   );
 
+  useEffect(() => {
+    setError({ recipient: '' });
+    if (mnsExtension.test(recipient)) {
+      const inputMns = recipient.replace(mnsExtension, '');
+      resolveDns(inputMns);
+    } else {
+      setMnsAddressCorrelation(false);
+      resetTargetMnsAddress();
+    }
+  }, [recipient]);
```

Comment:
React Hook useEffect has missing dependencies: 'mnsExtension', 'resetTargetMnsAddress', and 'resolveDns'. Either include them or remove the dependency array